test/langtools/tools/javac/SuperInit/LambdaOuterCapture.java

```diff
@@ -22,7 +22,7 @@
  */
 /*
  * @test
- * @bug 8194743
+ * @bug 8334252
```

Comment:
just a fix for the wrong bug ID

Reply:
In which case this also fixes JDK-8334679. If you want, add `/solves 8334679` here and I'll close PR #19816. Thanks.